tls/s2n_client_extensions.c

```diff
@@ -429,3 +445,17 @@ static int s2n_recv_client_sct_list(struct s2n_connection *conn, struct s2n_stuf
     /* Skip reading the extension, per RFC6962 (3.1.1) it SHOULD be empty anyway  */
     return 0;
 }
+
+
+static int s2n_recv_client_max_frag_len(struct s2n_connection *conn, struct s2n_stuffer *extension)
```

Comment:
I'm less worried about s2nd. Looks like you've already added an enable-mfl option to it. That's fine by me.